src/generate_transactions.rs

```diff
@@ -41,18 +54,27 @@ pub async fn send_continuous_transactions(
     Ok(())
 }
 
-async fn generate_and_send_transfer(tx_manager: &TransactionManager) -> Result<()> {
-    // Generate a new wallet for the recipient
-    let recipient_wallet = Wallet::new(&mut rand::thread_rng());
-    let to = recipient_wallet.address();
+async fn generate_and_send_transfer(
+    provider: Arc<Provider<Http>>,
+    mut tx_manager: TransactionManager,
+    level: u64,
+) -> Result<()> {
+    for i in (1..=level).rev() {
+        // Generate a new wallet for the recipient
+        let recipient_wallet = Wallet::new(&mut rand::thread_rng());
```

Comment:
This requires `CHAIN_ID`, which is set as a constant in `main.rs` (which isn't the best way to handle it 🤷‍♂️)